src/command_modules/azure-cli-configure/azure/cli/command_modules/configure/_consts.py

```diff
@@ -32,15 +32,15 @@
                ' $ az feedback\n'
 
 MSG_GLOBAL_SETTINGS_LOCATION = 'Your global settings can be found at {}'
-MSG_ACTIVE_ENV_SETTINGS_LOCATION = 'Your current environment settings can be found at {}'
+MSG_ACTIVE_CONTEXT_SETTINGS_LOCATION = 'Your current context settings can be found at {}'
```

Comment:
This sort of illustrates my point on "context". I have no idea what "context settings" means. However, if you took it out and it said "Your current settings can be found at..." that does make a lot more sense to me.